Firestore/core/src/remote/bloom_filter.cc

```diff
@@ -115,8 +115,30 @@ bool BloomFilter::MightContain(absl::string_view value) const {
 
 bool operator==(const BloomFilter& lhs, const BloomFilter& rhs) {
   return lhs.bit_count() == rhs.bit_count() &&
-         lhs.hash_count() == rhs.hash_count() &&
-         equal(lhs.bitmap().begin(), lhs.bitmap().end(), rhs.bitmap().begin());
+         lhs.hash_count() == rhs.hash_count() && CompareBits(lhs, rhs);
+}
+
+bool operator!=(const BloomFilter& lhs, const BloomFilter& rhs) {
```

Comment:
Put operator!= into the .h file, and declare it `inline`, like this:

https://github.com/firebase/firebase-ios-sdk/blob/b2383a9d3ca37b6a9e99c5fd9b75abdcf5a497c8/Firestore/core/src/model/transform_operation.h#L226-L229